File: fixtures/dart_output/post_with_escaped_double_quotes.dart
import 'package:http/http.dart' as http;

void main() async {
  var data = 'foo="bar"';

  var res = await http.post('http://example.com/', body: data);
  if (res.statusCode != 200) throw Exception('post error: statusCode= ${res.statusCode}');
  print(res.body);
}

File: fixtures/matlab_output/post_with_escaped_double_quotes.m
%% Web Access using Data Import and Export API
uri = 'http://example.com/';
body = 'foo="bar"';
response = webwrite(uri, body);

%% HTTP Interface
import matlab.net.*
import matlab.net.http.*
import matlab.net.http.io.*

uri = URI('http://example.com/');
body = FormProvider('foo', '"bar"');
response = RequestMessage('post', [], body).send(uri.EncodedURI);
